.github/ISSUE_TEMPLATE/change.yml

```
@@ -0,0 +1,44 @@
+name: "\U0001F4DD Request a change (not rule-related)"
+description: "Request a change that is not a bug fix, rule change, or new rule"
+title: "Change Request: (fill in)"
+labels:
+  - enhancement
+  - triage
+  - core
+body:
+- type: markdown
+  attributes:
+    value: ESLint adheres to the [Open JS Foundation Code of Conduct](https://eslint.org/conduct). By opening an issue, you agree to abide by the code of conduct.
+- type: input
+  attributes:
+    label: ESLint version
+    description: |
+      What version of ESLint are you currently using? (Run `npx eslint -v`.)
```

Purely a personal preference: I like to use long versions of CLI flags in scripts and documentation so the meaning is explicit even to those who aren't familiar with the tool or CLIs in general.

```suggestion
      What version of ESLint are you currently using? (Run `npx eslint --version`.)
```

---

Fair enough.